torch/_dynamo/variables/tensor.py

```diff
@@ -153,8 +153,19 @@ def var_getattr(self, tx, name):
         elif name == "data":
             result = self.call_method(tx, "detach", [], {})
         elif name == "T":
-            args = [variables.ConstantVariable(i) for i in range(self.ndim - 1, -1, -1)]
-            result = self.call_method(tx, "permute", args, {})
+            if self.ndim == 0:
+                result = self
+            else:
+                args = [
+                    variables.ConstantVariable(i) for i in range(self.ndim - 1, -1, -1)
+                ]
+                result = self.call_method(tx, "permute", args, {})
+        elif name == "mT" and self.ndim != 1:
+            if self.ndim == 0:
+                result = self
+            else:
+                args = [variables.ConstantVariable(i) for i in [-2, -1]]
+                result = self.call_method(tx, "transpose", args, {})
```

Comment:
So what ever happened to my original comment about using __get__

Reply:
sorry, don't think I saw that comment?

Reply:
The way this PR is constructed is you are manually desugaring the mH attribute into am equivalent series of transpose calls. The main problem with doing it this way is it's easy to do it wrong (dual source of truth) and the resulting program doesn't really capture what the previous program did. Ideally we would trace an mH access directly into the FX graph, however FX only supports function calls. The proposal was to make a function call with the getter function Tensor.mH.__get__. I vaguely remember someone telling me this didn't work but I'd quite like to know why it didn't work.